modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
+	@priority = 5
+	test AssertCRUDWithObjectDefinition {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object definition created") {
```

Comment:
`Given a custom object definition`

Remember that adjectives go before the noun in English, so the correct phrasing would be `created object definition`